src/server/main_service.cc

```diff
@@ -921,7 +920,7 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
     dfly_cntx->reply_builder()->CloseConnection();
   }
 
-  end_usec = ProactorBase::GetMonotonicTimeNs();
+  end_usec = ProactorBase::GetMonotonicTimeNs();  // misleading name, this is actually ns
```

Comment:
the name is misleading so lets fix it. call it end_ns, start_ns

Reply:
Definitely! I'll do some final touch-ups soon